graylog2-web-interface/src/stores/users/UsersStore.ts

```diff
@@ -95,6 +101,43 @@ export const UsersStore = {
 
     return promise;
   },
+
+  createToken(username: string, token_name: string): Promise<Token> {
+    const url = URLUtils.qualifyUrl(ApiRoutes.UsersApiController.create_token(encodeURIComponent(username),
+      encodeURIComponent(token_name), ).url);
+    const promise = fetch('POST', url);
+    return promise;
+  },
+
+  deleteToken(username: string, token_name: string): Promise<string[]> {
+    const  url = URLUtils.qualifyUrl(ApiRoutes.UsersApiController.delete_token(encodeURIComponent(username),
+      encodeURIComponent(token_name)).url, {});
+    const  promise = fetch('DELETE', url);
+
+    promise.then(() => {
+      UserNotification.success("Token \"" + token_name + "\" of user \"" + username + "\" was deleted successfully");
```

Comment:
Could we display in this notification the token name instead of the token itself?